SocketRocket/SRWebSocket.m

```diff
@@ -570,6 +570,33 @@ - (void)_initializeStreams;
     
     _inputStream.delegate = self;
     _outputStream.delegate = self;
+    
+    NSString *networkServiceType;
+    switch (_urlRequest.networkServiceType) {
+        case NSURLNetworkServiceTypeDefault:
+            return;
```

Comment:
A separate method could be better. 

I'm not sure about the safety of setting `networkServiceType` if it's break here. Because if the case is `NSURLNetworkServiceTypeDefault` it shouldn't set anything on the input & output stream.